synpurge/cli.py

```diff
@@ -115,6 +115,33 @@ def cleanup(path: "configuration file",
 
 
 @cmd
+def reindex(path: "configuration file",
+            concurrent: "enable enable concurrent reindexing" = False,
+            debug: "enable debugging output" = False,
+            verbose: "enable verbose operation" = False):
+    """Reindex the database after purging."""
```

Comment:
The command description should be just “Reindex the database”, because using “after purging” it gives the idea that the command both purges *and* reindexes, but it actually only reindexes.

---

On a different note: My first though was to have this in the `cleanup` subcommand, but I do actually think that this is good. Probably we should add a `vacuum` subcommand later on which does *only* vacuuming, and keep the existing `cleanup` which does both.